region/norfair/crocomire/Grapple Tutorial Room 3.json

```diff
@@ -111,13 +112,71 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway - Frozen Gamet",
+      "requires": [
+        "h_canNavigateUnderwater",
+        {"or": [
+          "h_canTrickyFrozenEnemyRunway",
+          {"and": [
+            "h_canFrozenEnemyRunway",
+            "Morph"
+          ]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 6,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "Lure the Gamets up out of the water. It may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
+        "Positioning is much easier with Morph; simply morph on the runway before they start separating.",
+        "Freeze a Gamet while there is a half-tile gap between it and the runway in order to extend it as much as possible."
+      ]
+    },
     {
       "link": [1, 1],
       "name": "Gamet Farm",
       "requires": [
         {"refill": ["Energy", "Missile", "Super", "PowerBomb"]}
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Leave With Runway - Frozen Gamet",
+      "requires": [
+        "h_canNavigateUnderwater",
+        "canTrickyJump",
+        {"or": [
+          "h_canTrickyFrozenEnemyRunway",
+          {"and": [
+            "h_canFrozenEnemyRunway",
+            "Morph"
+          ]}
+        ]},
+        {"or": [
+          {"ammo": {"type": "Super", "count": 1}},
+          {"obstaclesCleared": ["A"]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 9,
+          "openEnd": 0
+        }
+      },
+      "unlocksDoors": [
+        {"types": ["ammo"], "requires": []}
+      ],
+      "note": [
+        "Lure the Geemers up out of the water. It may be helpful to freeze them on the ascent to prevent them from separating and moving horizontally.",
+        "At the top, position a Gamet flush with the central runway. Positioning is much easier with Morph; simply morph on the runway before they start separating.",
+        "Lure the Gamet to the right and freeze it while there is a half-tile gap between it and the runway in order to extend it as much as possible."
```

Comment:
```suggestion
        "At the top, position a Gamet flush with the central runway. Positioning is much easier with Morph; simply morph on the runway before they start separating.",
        "Freeze the Gamet before it begins moving horizontally, and it will be able to turn to the right.",
        "Use it as a frozen platform to cross both bodies of water.",
        "Lure the Gamet to the right and freeze it while there is a half-tile gap between it and the runway in order to extend it as much as possible."
```